base/applications/notepad/lang/it-IT.rc

```diff
@@ -65,7 +65,7 @@ BEGIN
     BEGIN
         MENUITEM "&Guida in linea", CMD_HELP_CONTENTS
         MENUITEM SEPARATOR
-        MENUITEM "&About Notepad", CMD_HELP_ABOUT_NOTEPAD
+        MENUITEM "Informazioni su Blocco note", CMD_HELP_ABOUT_NOTEPAD
```

Comment:
do not strip the accelerator, use the one that 2k3 does use.

Reply:
I just checked and that is one that 2k3 uses:
![image](https://github.com/user-attachments/assets/909a3638-1928-435b-a564-bcb7e13ac8a1)
The only thing I missed is the '&s'. Gonna change it.

Reply:
resolved.